management/nodes_test.go

```diff
@@ -43,6 +43,27 @@ func TestNodeRegister(t *testing.T) {
 			}
 		})
 
+		t.Run("Reregister with different node name", func(t *testing.T) {
+			nodeName := pmmapitests.TestString(t, "node-name-for-all-fields")
+			nodeID, pmmAgentID := registerGenericNode(t, node.RegisterNodeBody{
+				NodeName: nodeName,
+				NodeType: pointer.ToString(node.RegisterNodeBodyNodeTypeGENERICNODE),
+				Address:  "node-address",
+				Region:   "region",
+			})
+
```

Comment:
Could you add a Registration of the same node with `Reregister: false` where we expect receiving an error from server

Reply:
Here I meant that you should do what you do, but for nodes with different name and same address region

Reply:
Sorry for unclear text